horizons/world/units/collectors/collector.py

```diff
@@ -396,8 +396,8 @@ def transfer_res_from_target(self):
 				new_reslist.append( entry )
 
 			remnant = self.get_component(StorageComponent).inventory.alter(entry.res, actual_amount)
-			assert remnant == 0, "%s couldn't take all of res %s; remnant: %s; planned: %s" % \
-			       (self, entry.res, remnant, entry.amount)
+			assert remnant == 0, "{} couldn't take all of res {}; remnant: {}; planned: {amount}".format(
```

Comment:
Why did you name make the last argument (`amount`) to format? Just use `{}` like the others.